lib/active_merchant/billing/gateways/adyen_checkout.rb

```diff
@@ -15,7 +15,7 @@ class AdyenCheckoutGateway < Gateway
       self.display_name = 'Adyen'
 
       PAYMENTS_API_VERSION = 'v51'
-      PAL_API_VERSION = 'v51'
+      PAL_API_VERSION = 'v49'
```

Comment:
What's the reason we're downgrading the version to v49?

Reply:
The reason is that the endpoint for disabling payment profile has the current version at 49 and for refunding has a current version at 51 and they both use the same base URL which is PAL_TEST_URL and PAL_API_VERSION. 


<img width="396" alt="Zrzut ekranu 2020-04-29 o 13 55 46" src="https://user-images.githubusercontent.com/567457/80593942-5f437900-8a22-11ea-814f-b3a5dae47aae.png">
<img width="398" alt="Zrzut ekranu 2020-04-29 o 13 55 52" src="https://user-images.githubusercontent.com/567457/80593945-6074a600-8a22-11ea-84a8-5293093ff480.png">
That's why I've decided to downgrade it to 49 because without this endpoint for disabling payment profile won't work, and refund still works on 49 version. If we want to to stay with the refund at the latest version then we will have to add another constant, something like this:
```
PAL_API_REFUND_VERSION = 51
PAL_API_DISABLE_VERSION = 49
```

Reply:
Looking at the url it seems these are two separate APIs: PAL Payment API and PAL Recurring API so perhaps it would be a good idea to treat the versions separately? 🤔